integration/auth/user_mgt_test.go

```diff
@@ -52,19 +64,33 @@ func TestUserManagement(t *testing.T) {
 func testCreateUsers(t *testing.T) {
 	// Create users with uid
 	for i := 0; i < 3; i++ {
-		params := (&auth.UserToCreate{}).UID(fmt.Sprintf("tempTestUserID-%d", i))
+		uid := fmt.Sprintf("tempTestUserID-%d", i)
+		params := (&auth.UserToCreate{}).UID(uid)
 		u, err := client.CreateUser(context.Background(), params)
 		if err != nil {
-			t.Fatal("failed to create user", i, err)
+			if strings.Contains(err.Error(), "DUPLICATE_LOCAL_ID") {
```

Comment:
Why?

Reply:
When running the integration tests, if they fail, or you exit them with CTRL-C, residual users on the backend should not prevent the tests from running if we want to create them anyway. 
This checks for the source of failure, and if the user already exists from the previous run, we let it slide.